tests/amor/pipeline_test.py

```diff
@@ -75,7 +78,53 @@ def test_run_full_pipeline(amor_pipeline: sciline.Pipeline):
     assert np.all(res.data[:, 1] >= 0)
 
 
+@pytest.mark.filterwarnings("ignore:Failed to convert .* into a transformation")
+@pytest.mark.filterwarnings("ignore:Invalid transformation, missing attribute")
+def test_pipeline_can_compute_reflectivity_merging_events_from_multiple_runs(
+    amor_pipeline: sciline.Pipeline,
+):
+    sample_runs = [
+        amor.data.amor_sample_run(608),
+        amor.data.amor_sample_run(609),
+    ]
+    pipeline = with_filenames(amor_pipeline, SampleRun, sample_runs)
+    pipeline[SampleRotation[SampleRun]] = pipeline.compute(
+        SampleRotation[SampleRun]
+    ) + sc.scalar(0.05, unit="deg")
+    result = pipeline.compute(ReflectivityOverQ)
+    assert result.dims == ('Q',)
+
+
+@pytest.mark.filterwarnings("ignore:Failed to convert .* into a transformation")
+@pytest.mark.filterwarnings("ignore:Invalid transformation, missing attribute")
+def test_pipeline_merging_events_result_unchanged(amor_pipeline: sciline.Pipeline):
+    sample_runs = [
+        amor.data.amor_sample_run(608),
+    ]
+    pipeline = with_filenames(amor_pipeline, SampleRun, sample_runs)
+    pipeline[SampleRotation[SampleRun]] = pipeline.compute(
+        SampleRotation[SampleRun]
+    ) + sc.scalar(0.05, unit="deg")
+    result = pipeline.compute(ReflectivityOverQ).hist()
+    sample_runs = [
+        amor.data.amor_sample_run(608),
+        amor.data.amor_sample_run(608),
+    ]
+    pipeline = with_filenames(amor_pipeline, SampleRun, sample_runs)
+    pipeline[SampleRotation[SampleRun]] = pipeline.compute(
+        SampleRotation[SampleRun]
+    ) + sc.scalar(0.05, unit="deg")
+    result2 = pipeline.compute(ReflectivityOverQ).hist()
+    assert_allclose(
+        2 * sc.values(result.data), sc.values(result2.data), rtol=sc.scalar(1e-6)
+    )
```

Comment:
It's not weird, for the reason you alluded to, the reflectivity is not normalized by the length of the run (or more precisely,  by the total intensity over the period of the run).
There's an issue for this here: #78 

Therefore there is currently generally no way for the workflow to distinguish the following two situations:

* There is a large number of events because the sample reflectivity is high.
* There is a large number of events because the run was longer than the reference run.

> this needs to change once reflectivity is normalized correctly

Yes that is correct.

Reply:
Isn't the name "reflectivity" misleading / wrong then?

Reply:
I don't think so. It's just that the data is not congruent with the model for how the data was produced.